evm/ts/__tests__/versions.test.ts

```diff
@@ -1,4 +1,4 @@
-import { Wormhole } from "@wormhole-foundation/sdk-connect";
+import { Wormhole } from "@wormhole-foundation/sdk";
```

Comment:
This one is fine since it runs in the context of tests, so it can safely import from `sdk`